decimal.go

```diff
@@ -902,6 +902,18 @@ func (d Decimal) RoundBank(places int32) Decimal {
 	return round
 }
 
+// RoundUp rounds decimal number towards positive infinite
+// (1.1).RoundUp(0) => 1
+// (1.52).RoundUp(1) => 1.6
+// (1.1001).RoundUp(2) => 1.11
+func (d Decimal) RoundUp(places int32) Decimal {
+	factor := NewFromFloat(10).Pow(NewFromInt(int64(places)))
```

Comment:
Decimal Pow is not a high-performance method. I have benchmarked them before but not have results here. You can try to benchmark it.

How about 
```go
factor := NewFromFloat(math.Pow(10, float64(-places))
```

Reply:
Thank you! Great suggestion, I'll try it out.

Reply:
using math.Pow:
```
BenchmarkDecimal_RoundUp-8                            	  799142	      1522 ns/op
```
using Decimal.Pow
```
BenchmarkDecimal_RoundUp-8                            	  255872	      4411 ns/op
```

I'll update the PR with your suggestion.